CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```

Comment:
```suggestion
CHANGELOG
```

Or, `Release History`

It just feels like someone's shouting.

Reply:
Can do

Reply:
Changed to 'Release History' since that is what it really is, w/out shouting